universum/configuration_support.py

```diff
@@ -40,10 +40,18 @@ class Step:
         will not be processed correctly and therefore should be splat into ``["ls", "-a"]``. Lists like
         ``["build.sh", "--platform A"]`` will not be processed correctly and thus should be plat into
         ``["build.sh", "--platform", "A"]``. A build step can have an empty list as a command - such step won't do
-        anything except showing up the step name in Universum execution logs. Some common actions, such as ``echo``
-        or ``cp``, are bash features and not actual programs to run. These features should be called as
-        ``["bash", "-c", "echo -e 'Some line goes here'"]``. Note that in this case the string to be passed to bash
-        is one argument containing white spaces and therefore not splat by commas.
+        anything except showing up the step name in Universum execution logs. Some common actions, such as ``echo``,
+        are bash features and not actual programs to run. These features should be called as
+        ``["bash", "-c", "echo -e 'Some line goes here'"]`` (or any other shell). Note that in this case
+        the string to be passed to bash is one argument containing white spaces and therefore not splat by commas.
+
+        .. note::
+
+            Python command line launching module is not entirely identical to usual shell. Arguments including
+            special characters (e.g. ``*.txt``) are treated as usual strings. To make the lack of flename expansion
+            (globbing) more obvious, these arguments are printed withing quotes in log. To use bash
```

Comment:
"withing" -> "within".
I would say, the real reason for using quotes is to make copy-pasting from logs easier.